verification/curator-service/ui/src/components/UploadsTable.tsx

```diff
@@ -170,12 +183,12 @@ class UploadsTable extends React.Component<Props, UploadsTableState> {
                                                         u.upload.summary
                                                             .numUpdated ?? 0,
                                                 };
-                                            },
-                                        );
+                                            });
                                         resolve({
                                             data: flattenedSources,
                                             page: query.page,
-                                            totalCount: result.data.total ?? 0,
+                                            totalCount:
+                                                flattenedSources.length ?? 0,
```

Comment:
This needs to remain as `result.data.total ?? 0`. This is the totalCount of all uploads in case there is more than one page. `flattenedSources` only contains the first page of results.